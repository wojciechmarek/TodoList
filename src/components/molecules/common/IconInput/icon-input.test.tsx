describe('IconInput', () => {
  it('should render', () => {
    expect(true).toBeTruthy();
  });

  describe('when the button is clicked', () => {});
});
