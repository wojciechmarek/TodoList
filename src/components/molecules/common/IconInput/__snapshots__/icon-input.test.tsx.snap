// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconButton should match the snapshot 1`] = `
.emotion-0 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  border-radius: var(--standard-border-radius);
  border: none;
  height: 3em;
}

.emotion-1 {
  border: 1px solid var(--search-input-border-and-icon-bg-color);
  background-color: var(--search-input-bg-color);
  color: var(--input-font-color);
  font-weight: 800;
  padding-left: 1em;
  border-radius: var(--standard-border-radius);
  -webkit-transition: filter 0.3s ease-in-out;
  transition: filter 0.3s ease-in-out;
  height: 100%;
  width: 100%;
  padding-right: 4em;
  box-shadow: var(--button-box-shadow);
}

.emotion-1::-webkit-input-placeholder {
  color: var(--text-font-color);
  font-weight: 600;
  opacity: 0.4;
}

.emotion-1::-moz-placeholder {
  color: var(--text-font-color);
  font-weight: 600;
  opacity: 0.4;
}

.emotion-1:-ms-input-placeholder {
  color: var(--text-font-color);
  font-weight: 600;
  opacity: 0.4;
}

.emotion-1::placeholder {
  color: var(--text-font-color);
  font-weight: 600;
  opacity: 0.4;
}

.emotion-1:hover {
  -webkit-filter: brightness(0.9);
  filter: brightness(0.9);
}

.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: absolute;
  right: 0.5em;
  height: 100%;
  width: 3em;
  border-radius: 0 var(--standard-border-radius);
  box-sizing: border-box;
  padding: 0.25em;
}

.emotion-2 svg {
  stroke: var(--search-magnifier-color);
}

.emotion-3 {
  height: calc(100% * 0.618);
  width: calc(100% * 0.618);
}

<div
  className="emotion-0"
>
  <input
    className="emotion-1"
    onChange={[Function]}
    placeholder="placeholder"
    value=""
  />
  <div
    className="emotion-2"
  >
    <div
      className="emotion-3"
    >
      <img
        alt="icon"
        src="https://via.placeholder.com/150"
      />
    </div>
  </div>
</div>
`;
